ob-pattern-1/templates/obam/configMaps/obam-bin.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: obam-bin
```

Comment:
@kdanW shall we stick to the resource and file naming conventions discussed in https://github.com/wso2/kubernetes-ei/issues/183?

Please feel free to drop in your suggestions and concerns, as well.

Reply:
I am +1 for the agreed upon convention. Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/2b2f6f4c2e0c8feada3a3eda00548a132db73a26